osx/KA-Lite-Monitor/KA-Lite-Monitor.xcodeproj/project.pbxproj

```diff
@@ -286,7 +286,7 @@
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_IDENTITY = "Foundation for Learning Equality, Inc.";
+				CODE_SIGN_IDENTITY = "";
```

Comment:
After a bit of testing on my side, I can now sign the .app file without the `codesign` command in `setup.sh`. This line just needs to be changed to `developer ID Application: Foundation for Learning Equality, Inc. (H83B64B6AV)`